massgov/openmass
# DP-21109: Add generic label feature to allow content tagging

**Description:**
Added field_reusable_label to the content types, configured form/view displays, created custom mg_labels metatag and mass_labels token.


**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-21109


**To Test:**
I would recommend to create a node of each type and see the page for mg_labels tag existence and visual issues.
